<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Podcasts_Add</class>
 <widget class="QDialog" name="Music_Podcasts_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Podcast</string>
  </property>
  <widget class="QLabel" name="AddPodcast_MainText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add New Podcast</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="AddPodcast_NumberText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Podcast Number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="AddPodcast_NumberBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddPodcast_KeyWordsText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Words</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPodcast_KeyWordsBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPodcast_CommentsBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddPodcast_CommentsText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPodcast_SaveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPodcast_CancelButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddPodcast_KeyValidation">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddPodcast_CommentsValidation">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddPodcast_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Podcasts_Add</receiver>
   <slot>AddPodcast()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddPodcast_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Podcasts_Add</receiver>
   <slot>CancelButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddPodcast()</slot>
  <slot>CancelButton()</slot>
 </slots>
</ui>
